frontend/src/App.vue

```diff
@@ -258,11 +259,11 @@ export default {
       this.toShowPromptMessage = true;
       this.promptMessageIcon = image;
 
-      if (type === "error") {
+      if (type === 0) {
```

Comment:
lets make sure the type remain string "Error" or "error"

Reply:
After we publish the new types version I will change it